packages/xo-web/src/common/xo/index.js

```diff
@@ -3185,6 +3185,9 @@ export const destroyProxyAppliances = proxies =>
 export const upgradeProxyAppliance = proxy =>
   _call('proxy.upgradeAppliance', { id: resolveId(proxy) })
 
+export const getProxyApplianceUpdates = proxy =>
+  _call('proxy.getApplianceUpdates', { id: resolveId(proxy) })
```

Comment:
I think we should slowly move away from `resolveId` where possible and explicitly require an id.